src/MunicipalityRegistry.Api.Extract/Extracts/ExtractFiles/ExtractFileName.cs

```diff
@@ -0,0 +1,65 @@
+namespace MunicipalityRegistry.Api.Extract.Extracts.ExtractFiles
```

Comment:
Should we move this entire thing to grar-common?

Reply:
To shaperon?